src/NuGet.Services.SearchService/Controllers/SearchController.cs

```diff
@@ -22,12 +22,15 @@ public class SearchController : ApiController
 
         private static readonly IReadOnlyDictionary<string, V2SortBy> SortBy = new Dictionary<string, V2SortBy>(StringComparer.OrdinalIgnoreCase)
         {
+            { "relevance", V2SortBy.Popularity},
```

Comment:
```suggestion
            { "relevance", V2SortBy.Popularity },
```

Reply:
IIRC this is not required due to the `switch` statement but perhaps this improves readability. The logic is now duplicated though.

Reply:
Yeah this was to improve readability since it took me by surprise in the beginning when I was trying to look for orderBy relevance and it had the "same" behaviour as orderBy "ajnhjfkbashsfbhsbf"